src/woo_publications/publications/tests/factories.py

```diff
@@ -18,3 +20,17 @@ class DocumentFactory(factory.django.DjangoModelFactory):
 
     class Meta:  # type: ignore
         model = Document
+
+    class Params:
+        document_service_configured = factory.Trait(
+            document_service=factory.SubFactory(
+                ServiceFactory,
+                api_root="https://example.com/",
+                api_type=APITypes.drc,
+                oas="https://example.com/api/v1/oas",
```

Comment:
no longer used, we disabled the UI aspect for this even in the settings.

Can you leave `oas`/`oas_file` empty, as to not cause confusion?